lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -715,14 +715,15 @@ handle_unary_op([$: | Rest], Line, Column, _Kind, Length, Op, Scope, Tokens) whe
   tokenize(Rest, Line, Column + Length + 1, Scope, [Token | Tokens]);
 
 handle_unary_op(Rest, Line, Column, Kind, Length, Op, Scope, Tokens) ->
-  case strip_horizontal_space(Rest, 0) of
-    {[$/ | _] = Remaining, Extra} ->
-      Token = {identifier, {Line, Column, nil}, Op},
-      tokenize(Remaining, Line, Column + Length + Extra, Scope, [Token | Tokens]);
-    {Remaining, Extra} ->
-      Token = {Kind, {Line, Column, nil}, Op},
-      tokenize(Remaining, Line, Column + Length + Extra, Scope, [Token | Tokens])
-  end.
+  Token = case {strip_horizontal_space(Rest, 0), Kind} of
+    {{[$/, $/ | _] = Remaining, Extra}, capture_op} ->
+      {capture_op, {Line, Column, nil}, Op};
+    {{[$/ | _] = Remaining, Extra}, _} ->
+      {identifier, {Line, Column, nil}, Op};
+    {{Remaining, Extra}, _} ->
+      {Kind, {Line, Column, nil}, Op}
+  end,
+  tokenize(Remaining, Line, Column + Length + Extra, Scope, [Token | Tokens]).
```

Comment:
```suggestion
  case {strip_horizontal_space(Rest, 0), Tokens} of
    {{[$/ | _] = Remaining, Extra}, [{capture_op, _, '&'} | _]} ->
      Token = {identifier, {Line, Column, nil}, Op},
      tokenize(Remaining, Line, Column + Length + Extra, Scope, [Token | Tokens]);
    {{Remaining, Extra}, _} ->
      Token = {Kind, {Line, Column, nil}, Op},
      tokenize(Remaining, Line, Column + Length + Extra, Scope, [Token | Tokens])
  end.
```

Reply:
Thanks a lot! :+1:

This works for all the tests you specified, however it seems to be breaking some [quote test doing `quote(do: + / 2)`](https://github.com/elixir-lang/elixir/blob/8770a61/lib/elixir/test/elixir/kernel/quote_test.exs#L267) though.
I can't figure if this is:
1. a legit case that got broken and needs to be addressed by an extra case
2. if this operation didn't make sense in the first place outside of a capture operator, and was relying on the current bug to work? (but maybe it is used by some metaprogramming somewhere?)

I went with 1. which seemed safer and tried a fix that I will push.